### ISSUE TITLE
moving messages to the DeltaChat folder not always reliable

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-core#654]
while the moving as such is much better and clearer with the last updates, there are sometimes some messages that are not moved for some reasons.

it's not clear to me yet under which circumstances messages are not moved, might be related to a multi-device setup, not sure.

any hints on this are warm welcome :)

**Comment by adbenitez** [comment]
I hate to say "I told you so" but I told you so ;)
https://github.com/deltachat/deltachat-core/issues/548

**Comment by adbenitez** [comment]
also sometimes I had noticed that messages stay in the inbox until I close Delta Chat and restart it again, others stay forever even after restarting the app

**Comment by r10s** [comment]
:)

however, i think, things are already much better than before #548, the remaining problem is not a general problem, the moving algorithm is quite clear now and that was not the case before. so, things are not 100% yet, but we're getting closer :)

**Comment by adbenitez** [comment]
yes, now most messages are moved, it is really much better now, 
in my case the main issue is that my network is really unstable, and some times "pure" dc messages are not moved to the INBOX/DeltaChat folder, seems like if something goes wrong while trying to move them, dc doesn't try again.
I guess people in fast networks doesn't even have this issue, I am actually amazed that you had experimented it.

**Comment by r10s** [comment]
> I guess people in fast networks doesn't even have this issue, I am actually amazed that you had experimented it.

in fact, i do not have it often, but i've also seen it :)
so, the hint with the flaky network might point to the correct direction for this issue :)
